ddtrace/internal/schema/__init__.py

```diff
@@ -21,6 +21,7 @@ def _validate_schema(version):
 _validate_schema(__schema_version)
 
 DEFAULT_SPAN_SERVICE_NAME = _DEFAULT_SPAN_SERVICE_NAMES[__schema_version]
+SCHEMA_VERSION = __schema_version
```

Comment:
nit: Instead of defining `__schema_version` can we use a helper function to retrieve and set the schema version

Reply:
We can use one to retrieve, but not to set.

The issue with setting the variable is the imported values are cached all over the place within the tracer, so even if someone (or something) changes the value after modules are loaded, you'll still get the original behavior.

I'll create a helper method for retrieving, though